avionics/common/src/sensor_collection.cpp

```diff
@@ -56,4 +58,9 @@ void SensorCollection::updateStatus() {
         raiseToStatus(status_, RocketStatus::NONCRITICAL_FAILURE);
         *status_bitfield_ |= 0x08;
     }
+    if (battery.getStatus() == SensorStatus::FAILURE) {
+        LOG_WARN("Voltage sensor failed");
+        raiseToStatus(status_, RocketStatus::NONCRITICAL_FAILURE);
+        *status_bitfield_ |= 0x04;
+    }
```

Comment:
Not critical now, but maybe as a TODO - it'd make sense to measure the voltage before launch and if it's too low, raise a critical or non-critical error (e.g. we could have a voltage level below which we shouldn't launch, and one that we could launch but it'd be non-ideal)

Reply:
TODO added and Trello created.

92fb4eb

https://trello.com/c/urowjTxS/375-check-voltage-level-before-launch